Xamarin.Forms.Platform.iOS/Renderers/SliderRenderer.cs

```diff
@@ -152,13 +157,47 @@ protected override void OnElementPropertyChanged(object sender, PropertyChangedE
 				UpdateThumbImage();
 			else if (e.PropertyName == Slider.ThumbColorProperty.PropertyName)
 				UpdateThumbColor();
+			else if (e.PropertyName == Specifics.UpdateOnTapProperty.PropertyName)
+				UpdateTapRecognizer();
 		}
 
 		void OnControlValueChanged(object sender, EventArgs eventArgs)
 		{
 			((IElementController)Element).SetValueFromRenderer(Slider.ValueProperty, Control.Value);
 		}
 
+		void UpdateTapRecognizer()
+		{
+			if (Element != null && Element.IsSet(Specifics.UpdateOnTapProperty))
+			{
+				if (Element.OnThisPlatform().GetUpdateOnTap())
+				{
+					if (_sliderTapRecognizer == null)
+					{
+						_sliderTapRecognizer = new UITapGestureRecognizer((recognizer) =>
+						{
+							var control = Control;
+							var tappedLocation = recognizer.LocationInView(control);
+							var val = (tappedLocation.X - control.Frame.X) * control.MaxValue / control.Frame.Size.Width;
```

Comment:
In the control gallery if you click really fast 5 times it causes the control gallery to reset which then causes a NRE here. It might be an edge case but adding a null check here on *CONTROL* would probably be a safe idea.

Reply:
@PureWeen Good catch. I've fixed this in new commit by adding null checks to `control` and `tappedLocation`. Seems like this happens because the control has already been removed when the last tap event is handled and this callback is called.